curriculum/js1/2.js

```diff
@@ -7,7 +7,15 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
+  if (a > b) { 
```

Comment:
You can make this easier by  doing both if(a > b && a > c){ return a}, this way you don't have to nest the if(a > c).